cmd/podman/common/build.go

```diff
@@ -453,6 +454,12 @@ func buildFlagsWrapperToOptions(c *cobra.Command, contextDir string, flags *Buil
 		runtimeFlags = append(runtimeFlags, "--systemd-cgroup")
 	}
 
+	configIndex := libpod.RuntimeNameToConfigIndex(podmanConfig.RuntimePath)
+
+	for _, arg := range podmanConfig.ContainersConfDefaultsRO.Engine.OCIRuntimesFlags[configIndex] {
+		runtimeFlags = append(runtimeFlags, "--"+arg)
```

Comment:
I don't think this is right, compared to podmanConfig.RuntimeFlags this will always append. I would assume if we set --runtime-flag on the cli it should overwrite the config file and not always add as append?